readme.md

```diff
@@ -0,0 +1,40 @@
+PAWS
+
+
+A Web Shell (PAWS) is a Jupyter notebooks deployment that has been customized to make interacting with Wikimedia wikis easier. It allows users to create and share documents that contain live code, visualizations such as graphs, rich text, etc. The user created notebooks are a powerful tool that enables data analysis and scientific research, and also transforms the way in which programmers write code - by enabling an exploratory environment with a quick feedback loop, and a low barrier for entry through it's easy to use graphical interface. 
+
+Quick Start
+
+git clone https://github.com/toolforge/paws.git
+
+cd paws
+
+AWS provides some very core features on top of which people can build stuff.
+Notebooks
+
+It provides Jupyter notebooks (previously known as IPython Notebooks)
+Web based Terminal
+Useful libraries
+Accessing Database Replicas With Pandas and Sqlalchemy
+
+Pandas is a lovely high level library for in-memory data manipulations. In order to get the result of a SQL query as a pandas dataframe use:
+
+from sqlalchemy import create_engine
```

Comment:
For this entire code block and any others, please format with this:
https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks
You can even have the python code syntax highlighted with that feature.